packages/design-system/src/components/N8nButton/Button.vue

```diff
@@ -75,6 +75,10 @@ export default Vue.extend({
 			validator: (value: string): boolean =>
 				['left', 'right'].includes(value),
 		},
+	  square: {
+		  type: Boolean,
+		  default: false,
+	  },
```

Comment:
why do we need a square prop? buttons are already now square...

Reply:
they are square only if the label is exactly one character, in the case of multiple characters if we want to keep the square shape we need a prop to differentiate

Reply:
I see.. can we delete `icon-button` class from here then and set `square` on IconButton component?